packages/client/src/views/CorrectionForm/CorrectionSummary.tsx

```diff
@@ -139,6 +139,12 @@ class CorrectionSummaryComponent extends React.Component<IFullProps, IState> {
         this.props.application.data
       )
     }
+    const currency = lookup.currencies({
+      code: window.config.CURRENCY && window.config.CURRENCY['isoCode']
+    })[0].symbol
+    ;(
+      this.group.fields[0].nestedFields as any
+    ).REQUIRED[0].label.defaultMessage += ` ${currency}`
```

Comment:
@sadmananik if you look carefully on your screen shot, when the currency is "K", it shows at the beginning of the amount but after changing that to "Bangladeshi Taka" the currency symbol is showing at the end that is not correct.